enterprise/api/v1/views.py

```diff
@@ -406,14 +441,21 @@ def license_revoke(self, request, *args, **kwargs):
         enrollments_by_course_id = self._enrollments_by_course_for_licensed_user(enterprise_customer_user)
 
         enrollment_api_client = EnrollmentApiClient()
+
+        revocation_results = {}
+        any_failures = False
         for course_overview in get_course_overviews(list(enrollments_by_course_id.keys())):
             enterprise_enrollment = enrollments_by_course_id.get(course_overview.get('id'))
+            course_id = str(course_overview.get('id'))
```

Comment:
nit: if you swap line 448 and 449, you could use `course_id` variable in `enrollments_by_course_id.get(course_overview.get('id'))`